appcircle_android_lint/1.0.1/component.yaml

```diff
@@ -0,0 +1,39 @@
+platform: Android
+buildPlatform: ReactNative, JavaKotlin
+displayName: Android Lint
+description: "Run lint gradle task on your Android project's source files."
+docUrl: "https://docs.appcircle.io/workflows/android-specific-workflow-steps/#android-lint"
+requiredComponents: "appcircle_git_clone"
+precedingComponents: "appcircle_activate_ssh_key"
+followingComponents:
```

Comment:
There is the output_folder variable in component's main.rb file, I think we can add export build artifact step.